recipes/backports.csv/meta.yaml

```diff
@@ -24,27 +20,11 @@ requirements:
   run:
     - python
 
-# test:
-  # Python imports
-  # imports:
-
-  # commands:
-    # You can put test commands to be run here.  Use this to test that the
-    # entry points work.
-
-
-  # You can also put a file called run_test.py in the recipe that will be run
-  # at test time.
-
-  # requires:
-    # Put any additional test requirements here.  For example
-    # - nose
+test:
+  imports:
+    - backports.csv
 
 about:
   home: https://github.com/ryanhiebert/backports.csv
-  license: UNKNOWN
+  license: Python Software Foundation License
```

Comment:
I guess we should stick to:
```
license: PSF
license_family: PSF
```
as done by conda-forge.

Reply:
I used the long version since `grep 'PSF\|Python Software Foundation' recipes/**/meta.yaml` did only output those -- though I'd also prefer `PSF` (or, even better, `PSF 2`)! But nevermind, I checked conda-forge's `python`, `argparse`, and, well..., `backports.csv` feedstocks and their identifiers are also not entirely consistent.
But the main thing is that `backports.csv` is in conda-forge and thus can be deleted here.